packages/synapse-react-client/src/synapse-queries/entity/useExportToTerra.test.tsx

```diff
@@ -0,0 +1,142 @@
+import { describe, it, expect, vi, beforeEach, Mock } from 'vitest'
+import { renderHook, act } from '@testing-library/react'
+import {
+  QueryBundleRequest,
+  SelectColumn,
+} from '@sage-bionetworks/synapse-types'
+import * as SynapseConstants from '@/utils/SynapseConstants'
+import { DEFAULT_PAGE_SIZE } from '@/utils/SynapseConstants'
+import { useExportToTerra } from './useExportToTerra'
+
+const mockCreatePfb = vi.fn()
+
+vi.mock('../table/useDownloadTable', () => ({
+  useDownloadTableQueryResultAsPFB: () => ({ mutateAsync: mockCreatePfb }),
+}))
+
+import { displayToast } from '../../components/ToastMessage/ToastMessage'
+vi.mock('../../components/ToastMessage/ToastMessage', () => ({
+  displayToast: vi.fn(),
+}))
+
+import { parseEntityIdFromSqlStatement } from '../../utils/functions/SqlFunctions'
+vi.mock('../../utils/functions/SqlFunctions', () => ({
+  parseEntityIdFromSqlStatement: vi.fn(),
+}))
+
+import { addDrsUriToSql } from './useExportToCavatica'
```

Comment:
surprised these `import`s don't need to be done before the instantiation of `mockCreatePfb`

Reply:
Moved imports